src/test/java/org/fundacionjala/coding/wilson/KataFizzBuzzTest.java

```diff
@@ -0,0 +1,82 @@
+package org.fundacionjala.coding.wilson;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class KataFizzBuzzTest {
+    /**
+     *
+     */
+    @Test
+    public void numberTestFizzBuzz() {
+        final int num = 15;
+        assertEquals("FizzBuzz", KataFizzBuzz.sendString(num));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "FizzBuzz" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)